agent/Cargo.toml

```diff
@@ -12,6 +12,8 @@ async-trait = "0.1.0"
 blake2 = "0.8.0"
 chrono = "0.4.10"
 cfg-if = "0.1"
+coap = { version = "0.9", optional = true }
+coap-lite = { version = "0.3", optional = true }
 env_logger = "0.6.1"
```

Comment:
I've added support to CoRE parsing in `coap-lite` 0.3.4, just published today. But `coap` is explicitly using `0.3.2` so I think that duplicates the dependency. I'll open a PR in `coap`.